examples/starter-tutorial/README.txt

```diff
@@ -0,0 +1,24 @@
+Starter Tutorial
+================
+
+Welcome to PyGMT! This tutorial is designed to teach the basic concepts to
+make a map in PyGMT.
+
+**About PyGMT**
+
+PyGMT is a Python wrapper for the Generic Mapping Tools (GMT) software package.
+GMT can be used for all types of plotting, but has a focus on scientific
+mapping in the geoscience field. PyGMT adds to the mapping capabilities of GMT
+by adding in the functionality of Python-specific packages, such as numpy,
+pandas, and xarray, as well as includes the use of Jupyter notebooks.
```

Comment:
We already have "About PyGMT" in the Overview page (https://www.pygmt.org/v0.5.0/overview.html). Do you think we can remove it from here?

Reply:
Should there still be any background info on PyGMT or should the tutorial index page just have links to the tutorials?

Reply:
I removed it in af13124